packages/hop-node/src/db/GasPricesDb.ts

```diff
@@ -51,7 +51,13 @@ class GasPricesDb extends BaseDb {
   }
 
   async getNearest (chain: string, targetTimestamp: number, staleCheck: boolean = true): Promise<GasPrice | null> {
-    const items : GasPrice[] = (await this.getItems()).filter((item: GasPrice) => item.chain === chain && item.timestamp)
+    const startTimestamp = targetTimestamp - OneHourSeconds
+    const endTimestamp = targetTimestamp + OneHourSeconds
+    const filter = {
+      gte: `${chain}:${startTimestamp}`,
+      lte: `${chain}:${endTimestamp}~`
```

Comment:
Is the tilde here intentional?

Reply:
yes it's intentional (mentioned above)

![DeepinScreenshot_select-area_20211004124338](https://user-images.githubusercontent.com/168240/135914324-20d3f2ad-97ed-447c-8347-520c2698fa72.png)

https://kevinsimper.medium.com/how-to-get-range-of-keys-in-leveldb-and-how-gt-and-lt-works-29a8f1e11782